tests/files/smbsecurityconfig2.yaml

```diff
@@ -18,7 +18,7 @@ metadata:
   name: adsec1
 spec:
   mode: active-directory
-  realm: zzz-best.x.asynchrono.us
+  realm: samba1.sink.test
```

Comment:
unrelated to this change, but is it correct to have this in lower case?

Reply:
Yes, the operator itself is responsible for any case conversions that are needed (if any).